### PR TITLE
URL does not get converted from snake_case responder_url to camelCase responderURL

### PR DESCRIPTION
**Pull request by Jared-Newell-Mobility**
see issue https://github.com/mobilityhouse/ocpp/issues/590

### CODE REVIEW

File: ocpp/charge_point.py [hunk 1]
```diff
@@ -54,6 +54,8 @@ def snake_to_camel_case(data):
         for key, value in data.items():
             key = key.replace("soc", "SoC")
             key = key.replace("_v2x", "V2X")
+            key = key.replace("_url", "URL")
```
Comment (proelke) [inline ocpp/charge_point.py:57]: The issue with this approach is another field, as I mentioned in my other PR. The ocppCsmsUrl field in a SetNetworkProfileRequest will fail due to this change. I would suggest adding test cases for any usage of URL in the spec.

Comment (OrangeTux) [reply ocpp/charge_point.py:57]: I believe these are all attributes that include `url`:

```
$ grep -i "url" ocpp/v201/{call,call_result,datatypes}.py
ocpp/v201/datatypes.py:    ocpp_csms_url: str
ocpp/v201/datatypes.py:    responder_url: str
```

**Review by OrangeTux — APPROVED** [review]

**Review by OrangeTux — CHANGES_REQUESTED** [review]
The tests don't pass. Can you fix them?

**Comment by Jared-Newell-Mobility** [comment]
This should now be complete - @proelke does this cover everything?

**Review by OrangeTux — APPROVED** [review]

**Pull request merged**